integration-testing/rnode_testing/docker.py

```diff
@@ -4,21 +4,21 @@
 
 
 def run_cmd(docker_container, cmd):
-    logging.info(f"{docker_container.name}: Execute <{cmd}>")
+    logging.info("{name}: Execute <{cmd}>".format(name=docker_container.name, cmd=cmd))
     r = docker_container.exec_run(['sh', '-c', cmd])
     output = r.output.decode('utf-8')
 
-    logging.info(f"{docker_container.name}: Finish <{cmd}>. Exit Code: {r.exit_code}")
+    logging.info("{name}: Finish <{cmd}>. Exit Code: {exit_code}".format(name=docker_container.name, cmd=cmd, exit_code=r.exit_code))
     return (r.exit_code, output)
 
 
 def list_containers(docker_client, network):
-    return docker_client.containers.list(all=True, filters={"name": f".{network}"})
+    return docker_client.containers.list(all=True, filters={"name": ".{network}".format(network=network)})
```

Comment:
Would you mind simplifying easy cases like this one to just `".{}".format(network)`?

Reply:
Yes.

Reply:
Ok~! I would make every string which contains only one format parameter in that way~!